docs/src/content/en/docs/rag/vector-databases.mdx

```diff
@@ -299,7 +323,7 @@ This helps users understand the constraints and avoid naming conflicts when crea
 The tabs provide examples of valid and invalid names to clarify the rules for each database.
 */}
 
-<Tabs items={['MongoDB', 'Pg Vector', 'Pinecone', 'Qdrant', 'Chroma', 'Astra', 'LibSQL', 'Upstash', 'Cloudflare', 'OpenSearch']}>
+<Tabs items={['MongoDB', 'Pg Vector', 'Pinecone', 'Qdrant', 'Chroma', 'Astra', 'LibSQL', 'Upstash', 'Cloudflare', 'OpenSearch', 'LanceDB']}>
```

Comment:
You added a tab for LanceDB here, but we just need a tab item to discuss it's naming rules.

Reply:
@NikAiyer I removed the tab for lancedb since I did not find any special naming conventions in the official docs.